corehq/apps/smsbillables/tests/utils.py

```diff
@@ -1,3 +1,38 @@
+import uuid
+from datetime import datetime
+
+from corehq.apps.sms.models import OUTGOING, SMS
+from corehq.apps.smsbillables.models import SmsGatewayFee
+
+short_text = "This is a test text message under 160 characters."
+
+long_text = (
```

Comment:
I was skimming this PR, read this and thought, "Sounds like @biyeun and @gherceg are really very kindred spirits" before realizing this code is pre-existing.

Reply:
It resonated deeply with me so wouldn't rule it out!

Reply:
🥬  and 🔈 for all!